### DISCUSSION TITLE
Multiple router connections to 1 VPS possible?

### DISCUSSION THREAD

**Discussion by peterxxl** [Ysurac/openmptcprouter#2667] [Q&A]
Hello

I am just getting started with this, but so far the experience has been awesome. I have 1 VPS and RPI 3B+ and bonding 4 connections - 2 Starlinks, 1 LTE and 1 geo stationary satellite connection. 

I have followed the posted guides and the setup was extremely easy and has been working great for me.

Now after the initial trial, I would like to expand the setup a little bit more - upgrading the Pi to 4B and another VPS.

I have few questions:
1 - when adding second VPS, this will act only as backup in case the first one fails? or will the traffic get distributed across both?
2 - after replacing the existing RBPi 3B+ with the more powerful 4B, I would like to setup the existing 3B at other location with different connections. My question is the following: can both Pies connect to the same 2 VPS' or do I need separate servers for each OMR router?

Thanks

**Comment by Ysurac** [comment]
1. Yes it's only a backup.
2. Not possible to connect multiple routers to same VPS.
